### PR TITLE
Feat mount path validation

### PR DESCRIPTION
**Pull request by wg102**
Closes https://github.com/StatCan/daaas-private/issues/71

### CODE REVIEW

File: frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+  ngOnChanges(): void {
+    this.initComponent();
+  }
+
+  private initComponent(): void{
+    // Get the list of mounted volumes of the existing Notebooks in the selected Namespace, AAW
+
+    this.volGroup.get('mount').setValidators([
+      Validators.required,
+      Validators.pattern(/^(((\/home\/jovyan)((\/)(.)*)?)|((\/opt\/openmpp)((\/)(.)*)?))$/)
```
Comment (mathis-marcotte) [inline frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts:26]: Is it openmpp or openmp? The regex says one while the error message says the other?

File: frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+  }
+  ngOnChanges(): void {
+    this.initComponent();
+  }
+
+  private initComponent(): void{
+    // Get the list of mounted volumes of the existing Notebooks in the selected Namespace, AAW
+
+    this.volGroup.get('mount').setValidators([
+      Validators.required,
```
Comment (mathis-marcotte) [inline frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts:25]: The red error box shows up for this required error, but theres no error message getting displayed. Probably need to add a mountName.hasError('required') condition to your message display function

File: frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts (hunk 1 above)
Comment (wg102) [reply frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts:26]: openmpp, fixed

**Review by mathis-marcotte — APPROVED** [review]
LGTM

File: frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
   @Input() volGroup: FormGroup;
 
   constructor() {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.initComponent();
+  }
+  ngOnChanges(): void {
+    this.initComponent();
```
Comment (mathis-marcotte) [inline frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts:18]: Is this needed to be also in the onChanges function? Shouldnt it be enough to just be in the onInit function?

Comment (wg102) [reply frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts:18]: It is to force the validation when text is changed.

**Pull request merged**